src/main/pages/che-7/overview/assembly_installing-che-on-openshift-3-using-the-operator.adoc

```diff
@@ -29,6 +29,8 @@ Operators are a method of packaging, deploying, and managing a Kubernetes applic
 
 This chapter describes how to install {prod-short} on OpenShift 3, with the CLI management tool, using the Operator method.
 
+NOTE: This approach is only supported for use with {ocp} and {osd} version 3.11, but may also work for newer versions of {ocp} and {osd}.
```

Comment:
Don't use 'may'. It either works, or not. I thought we were treating it as a back up method.